deps/lz4/Makefile

```
@@ -0,0 +1,29 @@
+
+R_CC=$(CC) $(R_CFLAGS)
+R_LD=$(CC) $(R_LDFLAGS)
+
+AR ?= ar
+ARFLAGS= rcs
+
+liblz4.a: lz4.o lz4hc.o lz4frame.o xxhash.o
+	$(AR) $(ARFLAGS) $@ $^
+
+lz4.o: lz4.c lz4.h
+lz4hc.o: lz4hc.c lz4hc.h lz4.h
```

checkmake (0.3.2)

[warning] Target "lz4hc.o" should be declared PHONY.

(phonydeclared)